### PR TITLE
Add persistent storage

### PR DESCRIPTION
**Pull request by livingshade**
TODO: 
- different storage for different version of review
- products

### CODE REVIEW

File: kubernetes/apply/details/details-mongodb.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+
+# mongodb service and deployment
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    kompose.cmd: kompose convert
```
Comment (Romero027) [inline kubernetes/apply/details/details-mongodb.yaml]: Please remove these unused kcompose annotations

**Review by Romero027 — CHANGES_REQUESTED** [review]
Can you merge all yaml files into a single yaml file? Ideally, we want to deploy it with just one command.

And please remove all kcompose annotations. 

Thanks!

**Comment by livingshade** [comment]
> Please remove these unused kcompose annotations

Done

> Can you merge all yaml files into a single yaml file? Ideally, we want to deploy it with just one command.

A single large yaml file might be a little bit too complex? I have put them together into a folder and a single `kubectl apply -Rf ./kubernetes/apply` will do.

**Comment by Romero027** [comment]
looks good. Thanks

**Pull request merged**